ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -535,26 +816,110 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: baremetal-lab-agent-install-network-dynamic-day2
-- as: baremetal-compact-agent-ipv4-dhcp-disc-day2-amd-mixarch-f28
+- as: baremetal-ipi-ovn-ipv4-amd-f28
   cluster: build05
-  cron: 36 13 7 * *
+  cron: 11 20 9 * *
   steps:
     cluster_profile: equinix-ocp-metal-qe
     env:
-      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
-      ADDITIONAL_WORKERS: "1"
       AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
-      DISCONNECTED: "true"
+      DISCONNECTED: "false"
       RESERVE_BOOTSTRAP: "false"
-      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
       architecture: amd64
-      ipv4_enabled: "true"
-      ipv6_enabled: "false"
       masters: "3"
-      workers: "0"
+      workers: "2"
     test:
-    - chain: openshift-upgrade-qe-test-disconnected
-    workflow: baremetal-lab-agent-install-network-dynamic-disconnected-day2
+    - chain: openshift-upgrade-qe-test
+    workflow: baremetal-lab-ipi
+- as: baremetal-ipi-ovn-ipv4-amd-vmedia-f28
+  cluster: build05
+  cron: 5 15 12 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      masters: "3"
+      workers: "2"
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: baremetal-lab-ipi-virtual-media
+- as: baremetal-ipi-ovn-ipv4-arm-f28
```

Comment:
This one is duplicated with `baremetal-ipi-ovn-ipv4` ?

Reply:
If I remember correctly, we agree to add both `-amd` and `-arm` for a profile that make sure we don't have jobs in single-arch config file. And in e2e config file there are also 2 jobs: `baremetal-ipi-ovn-ipv4-amd-f7` and `baremetal-ipi-ovn-ipv4-arm-f7`. So I will keep this one.

Reply:
> And in e2e config file there are also 2 jobs

Actually other platforms also test both arm and amd, but be different with baremetal, generally there are two jobs, one e2e ci job, another is e2e destructive ci job, we use one to cover amd, another to cover arm for cross coverage. If we always take cpu-arch into consideration strictly, This comes to the same discussion in https://github.com/openshift/release/pull/54740#discussion_r1699324320 .

Reply:
Not exact same with https://github.com/openshift/release/pull/54740#discussion_r1699324320. Let's eave the decision to @aleskandro.

Reply:
@aleskandro is also on PTO, I will remove `baremetal-ipi-ovn-ipv4-amd-f28`, if @aleskandro confirm we need it, I will add it back. WDYT @jianlinliu ?